api/pyproject.toml

```diff
@@ -47,9 +47,10 @@ ignore = [
     "B006", # mutable-argument-default
     "B007", # unused-loop-control-variable
     "B026", # star-arg-unpacking-after-keyword-arg
-#    "B901", # return-in-generator
     "B904", # raise-without-from-inside-except
     "B905", # zip-without-explicit-strict
+    "N806", # non-lowercase-variable-in-function
+    "N815", # mixed-case-variable-in-class-scope
```

Comment:
Some clarification for these exclued rules:
- N806 is too strict to enforcing all the variable names in lowercase. We have common usage in getting parameters in tool parameters, which often contains CapWord style.
- N815 is excluded as we have `en_US` style available in classes.